web_access_rule_buttons/static/src/js/web_access_rule_buttons.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * © 2016 Camptocamp SA
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+ */
+odoo.define('web_access_rule_buttons.main', function (require) {
+  "use strict";
+
+  var core = require('web.core');
+
+  var FormView = require('web.FormView');
+  var ListView = require('web.ListView');
+
+  var QWeb = core.qweb;
+
+  var _t = core._t;
```

Comment:
I know we are a lot to leave those 2 lines in the beginning of the class but I still wondering why we are setting them if we are not using them?